package.json

```diff
@@ -17,27 +17,35 @@
     "postinstall": "rn-nodeify --install stream,process,path,events,crypto,console,buffer --hack"
   },
   "dependencies": {
+    "@ethersproject/shims": "5.7.0",
     "@fortawesome/fontawesome-svg-core": "1.2.36",
     "@fortawesome/free-regular-svg-icons": "6.4.0",
     "@fortawesome/free-solid-svg-icons": "6.4.0",
     "@fortawesome/react-native-fontawesome": "0.2.7",
-    "@hathor/hathor-rpc-handler": "0.0.1-experimental-alpha",
+    "@hathor/hathor-rpc-handler": "^0.0.1-experimental-alpha",
```

Comment:
Why?

Reply:
We should update it before merging this PR